Resources/config/services.yml

```diff
@@ -12,7 +12,7 @@ services:
       class: 'Enqueue\ElasticaBundle\Doctrine\Queue\SyncIndexWithObjectChangeProcessor'
       public: true
       arguments:
-          - '@doctrine'
+          - null
```

Comment:
could you please add a comment that the right manager will be injected in the extension